Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -1015,12 +1014,16 @@ public void executeDmsExport(Task step, boolean automatic) throws DataException,
     }
 
     /**
-     * Get open tasks for current user sorted according to sort query.
-     * @param sort possible sort query according to which results will be sorted
+     * <<<<<<< HEAD Get open tasks for current user sorted according to sort query.
```

Comment:
Looks like a merged conflict on rebasing this branch. Please adjust the JavaDoc.

Reply:
done